mysticeti-core/src/types.rs

```diff
@@ -69,6 +69,25 @@ impl Hash for BlockReference {
     }
 }
 
+#[derive(Clone, Copy, Eq, PartialEq, Serialize, Deserialize, Default)]
+pub struct AuthorityRound {
```

Comment:
Should we add a method `into_inner` and re-use that structure elsewhere in the committer? I remember there are many places where we use tuples of (round, authority) and I was alway afraid to swap them, ie. use (authority, round) instead of (round, authority)